terraform/environments/production/relays.tf

```diff
@@ -384,34 +384,6 @@ module "relays" {
   token   = var.relay_token
 }
 
-# Allow SSH access using IAP for relays
```

Comment:
Does this mean we won't have SSH access to the Relays at all anymore?

Reply:
Nah - this was a little bit of a mess. This is just moving the SSH config to the `main` module with the SSH rules for the other instances.

Reply:
Ah, I see.  Apologies for missing that.